services/harbor/1.16.0/valkey/valkey.yaml

```diff
@@ -31,3 +31,14 @@ spec:
     - kind: ConfigMap
       name: harbor-valkey-overrides
       optional: true
+  postRenderers:
```

Comment:
@mhrabovcin why not directly use the `metrics.service.annotations` value in the helm chart ??

https://github.com/mesosphere/charts/blob/10f659401f8163ac03d8e811c3e29b00181b83b5/stable/valkey/templates/metrics-svc.yaml#L15

Reply:
I think that `ServiceMonitor` selects endpoints by labels, `annotations` wouldn't work here.

Reply:
agree missread the sevice tempalte in valkey chart
only solution is `commonLabels` in the value , but will change it on all object